cea/interfaces/dashboard/lib/database/models.py

```
@@ -130,22 +130,27 @@ async def migrate_db():
     # TODO: Remove once in release new version
     # Check and update existing table schemas
     engine = get_engine()
-    with engine.connect() as conn:
-        inspector = inspect(engine)
+    async with engine.connect() as conn:
+        # Use run_sync to execute synchronous inspection operations
+        inspector = await conn.run_sync(lambda sync_conn: inspect(sync_conn))
```

Ruff (0.11.9)

Local variable `inspector` is assigned to but never used

Remove assignment to unused variable `inspector`

(F841)